elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQueryResult.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.async.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+
+import com.yahoo.elide.annotation.CreatePermission;
+import com.yahoo.elide.annotation.Exclude;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.annotation.UpdatePermission;
+
+/**
+ * Model for Async Query Result
+ */
+@Entity
+@Include(type="queryResult")
+@ReadPermission(expression = "Principal is Owner")
+@UpdatePermission(expression = "Prefab.Role.None")
+@CreatePermission(expression = "Prefab.Role.None")
+public class AsyncQueryResult implements PrincipalOwned {
```

Comment:
What about DeletePermission

Reply:
Should we set the DeletePermission as None for the AsyncQueryResult ?